fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AnalyzeStmtAnalyzer.java

```diff
@@ -130,13 +130,33 @@ public Void visitCreateAnalyzeJobStatement(CreateAnalyzeJobStmt statement, Conne
                 if ((Strings.isNullOrEmpty(tbl.getCatalog()) &&
                         CatalogMgr.isExternalCatalog(session.getCurrentCatalog())) ||
                         CatalogMgr.isExternalCatalog(tbl.getCatalog())) {
-                    throw new SemanticException("External Table don't support analyze job");
+                    if (tbl.getTbl() == null) {
```

Comment:
it's better to normalizationTableName first

Reply:
We can normalize the tableName, because it's tbl could be null.